config/manager/default-config/secret-envs.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: backend-auth-secret
+stringData:
+  # generated with the command below (from https://janus-idp.io/docs/auth/service-to-service-auth/#setup):
+  # node -p 'require("crypto").randomBytes(24).toString("base64")'
+  BACKEND_SECRET: "R2FxRVNrcmwzYzhhN3l0V1VRcnQ3L1pLT09WaVhDNUEK" # notsecret
```

Comment:
We should not use a default credentials here.

Reply:
it is a matter of configuration, secret-envs is not mandatory or can be created as configmap-envs (like it managed in the  main branch https://github.com/janus-idp/operator/blob/main/config/manager/default-config/backend-auth-configmap.yaml)